packages/api/src/SubmittableExtrinsic.ts

```diff
@@ -76,11 +85,11 @@ export interface SubmittableExtrinsic<ApiType> extends IExtrinsic {
 
   sign (account: IKeyringPair, _options: Partial<SignatureOptions>): this;
 
-  signAndSend (account: IKeyringPair | string | AccountId | Address, options?: Partial<SignatureOptions>): SumbitableResultResult<ApiType>;
+  signAndSend (account: IKeyringPair | string | AccountId | Address, options?: Partial<SignerOptions>): SumbitableResultResult<ApiType>;
```

Comment:
Two lines above it's still `SignatureOptions`, I guess SignerOptions is more flexible there too

Reply:
Yes, cannot quite change the sign signature, which is a muck-up - since that sits in `IExtrinsic`. Had it changed and then it complains bitterly.

Reply:
Ah i see what you mean, ok